proposals/0452-integer-generic-parameters.md

```diff
@@ -54,11 +54,29 @@ Or it can be instantiated using integer generic parameters from the surrounding
 generic environment:
 
 ```swift
-struct Matrix<let n: Int, let m: Int> {
-    var matrix: Vector<n, Vector<m, Double>>
+struct Matrix<let columns: Int, let rows: Int> {
+    var matrix: Vector<columns, Vector<rows, Double>>
 }
 ```
 
+Integer generic parameters become static constant members of the type, with
+the same visibility as the type itself:
+
+```swift 
+public struct Matrix<let columns: Int, let rows: Int> {
+    // implicitly has these members:
+    //  public static let columns: Int
+    //  public static let row: Int
```

Comment:
Typo: row(s). Should these be *computed* properties? Static stored properties aren't supported in generic types.

```suggestion
    //  public static var columns: Int { get }
    //  public static var rows: Int { get }
```